src/controllers/profile.controller.js

```diff
@@ -0,0 +1,115 @@
+import bcrypt from 'bcrypt';
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../models/user.js';
+import { profileService } from '../services/profile.service.js';
+import { emailService } from '../services/email.service.js';
+
+const getAllInfo = async (req, res) => {
+  const users = await profileService.getAllInfo();
+
+  res.send(users.map(profileService.normalize));
+};
+
+const getProfile = async (req, res) => {
+  const user = await User.findOne({ where: { id: req.user.id } });
+
+  if (!user) {
+    throw ApiError.unauthorized('User not found');
+  }
+  res.send(profileService.normalize(user));
+};
+
+const changeName = async (req, res) => {
+  const { newName } = req.body;
+  const userId = req.userId;
+
+  if (!newName) {
+    throw ApiError.badRequest('Enter new name');
+  }
+
+  const user = await User.findOne({ where: { userId } });
```

Comment:
The query should use 'id' instead of 'userId' in the where clause. Ensure that the correct field is being used to find the user.